src/containers/pages/JurisdictionAssignment/JurisdictionTable/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -48,3 +48,164 @@ exports[`src/containers/pages/jurisdictionView/jurisdictionTable works correctly
   Akros_1
 </span>
 `;
+
+exports[`src/containers/pages/jurisdictionView/jurisdictionTable works correctly through a full render cycle: structure count rendered 1`] = `
```

Comment:
This snapshot is too large for merely showing us that we get a structure count.  Please consult: https://kentcdodds.com/blog/effective-snapshot-testing

Idea: maybe just have a snapshot of the text, or maybe just assert the props match some expected value